test-case/multiple-pause-resume.sh

```diff
@@ -135,6 +135,13 @@ expect {
         if { \$i > $repeat_count } { exit 0 }
         exp_continue
     }
+    "*#*+*MAX" {
+        set sleep_t [expr int([expr rand() * $rnd_range]) + $rnd_min ]
```

Comment:
Do our tests generally have a lot of randomness?  Doesn't that make failures less deterministic?

Reply:
In general we don't like randomness in testing. In this pause/resume, we intentionally introduce randomness on ranger of interval to discover easily fw bug